site/public/js/user-profile.js

```diff
@@ -39,6 +39,20 @@ function getAvailableTimeZones() {
     return $('#time_zone_selector_label').data('available_time_zones').split(',')
 }
 
+/**
```

Comment:
Generally good to leave a link to the source of where a code snippet was from in the doccomments 

It looks like this is from https://stackoverflow.com/questions/9149556/how-to-get-utc-offset-in-javascript-analog-of-timezoneinfo-getutcoffset-in-c

Reply:
Done!